astronomer/providers/google/cloud/example_dags/example_bigquery_sensors.py

```diff
@@ -0,0 +1,97 @@
+"""Example Airflow DAG for Google BigQuery Sensors."""
+import os
+from datetime import datetime
+
+from airflow import models
+from airflow.providers.google.cloud.operators.bigquery import (
+    BigQueryCreateEmptyDatasetOperator,
+    BigQueryCreateEmptyTableOperator,
+    BigQueryDeleteDatasetOperator,
+    BigQueryInsertJobOperator,
+)
+from airflow.providers.google.cloud.sensors.bigquery import (
+    BigQueryTablePartitionExistenceSensor,
+)
+
+from astronomer.providers.google.cloud.sensors.bigquery import (
+    BigQueryTableExistenceSensorAsync,
+)
+
+PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "astronomer-airflow-providers")
+DATASET_NAME = os.environ.get("GCP_BIGQUERY_DATASET_NAME", "astro_dataset")
+LOCATION = os.environ.get("GCP_LOCATION", "us")
```

Comment:
I think we also need a GCP connection id. we should add it as env here and use it in either default param or require operator otherwise auth will fail if the connection name will be not default.

Reply:
Added `GCP_CONN_ID = os.environ.get("GCP_CONN_ID", "google_cloud_default")`  as os env variables